src/app/WriteClient.h

```diff
@@ -39,6 +39,8 @@ namespace app {
 /**
  *  @brief The read client represents the initiator side of a Write Interaction, and is responsible
  *  for generating one Write Request for a particular set of attributes, and handling the Write response.
+ *  Consumer can allocate one write client, then call PrepareAttribute followed by FinishAttribute for every attribute
+ *  it want to insert in write request, then call SendWriteRequest
```

Comment:
```suggestion
 *  it wants to insert in write request, then call SendWriteRequest
```

But actually, the consumer will need to insert the values too, no?

Reply:
Done, update with comments, yes, insert the value in write request.